setup.py

```diff
@@ -9,6 +9,7 @@
     'flask',
     'pyyaml',
     'requests',
+    'mohawk',
```

Comment:
Nitpicky, but this list is best if ordered alphabetically ;) Insert mohawk after flask.